upgrade-tidb-using-tiup.md

```diff
@@ -166,55 +188,49 @@ tiup cluster display <cluster-name>
+TiDB Version: v4.0.0
 ```
 
 ## FAQ
 
 This section describes common problems encountered when updating the TiDB cluster using TiUP.
 
-### If an error occurs and the updated is interrupted, how to resume the update from the point of the interruption after fixing this error?
+### If an error occurs and the upgrade is interrupted, how to resume the upgrade after fixing this error?
 
-You can specify `--role` or `--node` to update the specified component or node. Here is the command:
+Re-execute the `tiup cluster upgrade` command to resume the upgrade. The upgrade operation restarts the nodes that have been previously upgraded. In subsequent 4.0 versions, TiDB will support resuming the upgrade from the interrupted point.
 
-{{< copyable "shell-regular" >}}
+### The evict leader has waited too long during the upgrade. How to skip this step for a quick upgrade?
 
-```shell
-tiup cluster upgrade <cluster-name> v4.0.0-rc --role tidb
-```
-
-or
-
-{{< copyable "shell-regular" >}}
-
-```shell
-tiup cluster upgrade <cluster-name> v4.0.0-rc --node <ID>
-```
-
-### The evict leader has waited too long during the update. How to skip this step for a quick update?
-
-You can specify `--force`. Then the processes of transferring PD leader and evicting TiKV leader are skipped during the update. The cluster is directly restarted to update the version, which has a great impact on the cluster that runs online. Here is the command:
+You can specify `--force`. Then the processes of transferring PD leader and evicting TiKV leader are skipped during the upgrade. The cluster is directly restarted to update the version, which has a great impact on the cluster that runs online. Here is the command:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-tiup cluster upgrade <cluster-name> v4.0.0-rc --force
+tiup cluster upgrade <cluster-name> v4.0.0 --force
 ```
 
-### How to update the version of tools such as pd-ctl after updating the TiDB cluster?
+### How to update the version of tools such as pd-ctl after upgrading the TiDB cluster?
 
-Currently, TiUP does not update and manage the version of tools. If you need the tool package of the latest version, directly download the TiDB package and replace `{version}` with the corresponding version such as `v4.0.0-rc`. Here is the download address:
+Currently, TiUP does not update and manage the version of tools. If you need the tool package of the latest version, directly download the TiDB package and replace `{version}` with the corresponding version such as `v4.0.0`. Here is the download address:
 
 {{< copyable "" >}}
 
 ```
 https://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz
 ```
 
+### Failure to upgrade the TiFlash component during the cluster upgrade
+
+Before v4.0.0-rc.2, TiFlash might have some incompatibility issues. This might cause problems when you upgrade a cluster that includes the TiFlash component to v4.0.0-rc.2 or a later version. If so, go to [ASK TUG](https://asktug.com/) and ask for R&D support.
```

Comment:
```suggestion
Before v4.0.0-rc.2, TiFlash might have some incompatibility issues. This might cause problems when you upgrade a cluster that includes the TiFlash component to v4.0.0-rc.2 or a later version. If so, [contact the R&D support](mailto:support@pingcap.com).
```